### DISCUSSION TITLE
Combine People and socials

### DISCUSSION THREAD

**Discussion by redimongo** [Podcastindex-org/podcast-namespace#523] [Enhancement Proposal]
So I was going through the code for the example rss feed over at 
https://github.com/Podcastindex-org/podcast-namespace/blob/main/example.xml

What I noticed strange was how we have podcast:person and also podcast:socialInteract (see below)
```
 <item>
            <podcast:person href="https://www.podchaser.com/creators/adam-curry-107ZzmWE5f" img="https://example.com/images/adamcurry.jpg">Adam Curry</podcast:person>
            <podcast:person role="guest" href="https://github.com/daveajones/" img="https://example.com/images/davejones.jpg">Dave Jones</podcast:person>
            <podcast:person group="visuals" role="cover art designer" href="https://example.com/artist/beckysmith">Becky Smith</podcast:person>

            <podcast:socialInteract priority="1" uri="https://podcastindex.social/web/@dave/108013847520053258" protocol="activitypub" accountId="@dave" accountUrl="https://podcastindex.social/web/@dave" />
            <podcast:socialInteract priority="2" uri="https://twitter.com/PodcastindexOrg/status/1507120226361647115" protocol="twitter" accountId="@podcastindexorg" accountUrl="https://twitter.com/PodcastindexOrg" />

        </item>

```

However we don't seem to either link the socialInteract to the person or person to socialInteract

Imagine an app that had it so that a lower 3rd was shown on the screen or even a button that said `People In Episode` While it can still be done, it would take the app developer to map and store all these social links and work out away to match it to a person.

Instead I would like to suggest we integrate these a little more like this:

I'll use Dave as the example:

Before
```
<podcast:person role="guest" href="https://github.com/daveajones/" img="https://example.com/images/davejones.jpg">Dave Jones</podcast:person>
```

After:
```
<podcast:person role="guest" href="https://github.com/daveajones/"
img="https://example.com/images/davejones.jpg" name="Dave Jones">
  <podcast:link rel="youtube" href="https://www.youtube.com/channel/UC2mg7BPUowFdqhekg3GFEKA" />
  <podcast:link rel="twitter" href="https://twitter.com/daveajones" />
  <podcast:link rel="facebook" href="https://www.facebook.com/daveajones" />
  <podcast:link rel="website" href="https://daveajones.com" />
</podcast:person>
```

I know it would replace the `podcast:socialInteract` but from a database management it would allow podcast hosting providers to offer a better experience to the public if these two are joined  you could still utilise `podcast:socialInteract ` as a method to interact with the episode or something.

I know these would be huge changes but one, I think could potentially be for the better.

**Comment by picard102** [comment]
Isn't the socialInteract tag for linking to specific social media posts, rather than users?

**Reply by redimongo** [reply]
It could be, but that's not the first thought I get when I see the word SocialInteract. 

Social interact is about interacting with the show. 

Ideally your right it should have things like `timestamp` which then could allow a podcast app to show a card with that information at the correct time.

**Reply by picard102** [reply]
> https://github.com/Podcastindex-org/podcast-namespace/blob/main/docs/1.0.md#social-interact 
> The socialInteract tag allows a podcaster to attach the url of a "root post" of a comment thread to an episode. 

Maybe there would be more than one comment thread per episode, but I'm not sure that was the intent.

**Reply by nathangathright** [reply]
The socialInteract tag can support more than one comment thread per episode, which makes perfect sense for support multiple protocols (i.e. twitter and activitypub). Any efforts to provide multiple links per person should have nothing to do with the socialInteract tag.

**Comment by samsethi** [comment]
Hi @redimongo 

I think this is a smart idea to combine person and social.  We already do this inside of Podfans.  I would love to have this in the RSS field and auto-populate the social fields in our host's profile from the RSS. 

However it is not combining the socialInteract tag as that is used for the location of comments.  But simply combining them inside the person tag would be great.  I know Daniel J Lewis would also like to extend the use of the person tag to include the podcasters role and avatar. 

We already have all of these fields but podfan users have to manually enter the details.  

<img width="456" alt="Podfans___Dashboard" src="https://github.com/Podcastindex-org/podcast-namespace/assets/3492066/f4c9480b-ff05-4944-9009-25fc98c5fbdb">

**Reply by redimongo** [reply]
@samsethi What does your RSS feeds look like, we do need to make sure we standardise these things. Otherwise it's the wild Wild West. 

We need rules lol 😂

**Reply by nathangathright** [reply]
> I know Daniel J Lewis would also like to extend the use of the person tag to include the podcasters role and avatar.

[Person](https://github.com/Podcastindex-org/podcast-namespace/blob/main/docs/1.0.md#person) already supports role and image?
